examples/getSourceSet.py

```diff
@@ -327,20 +338,24 @@ def showStandards(standardStarSet, exp, frame, countsMin=None, flagMask=None, rm
             pt = "+"
         ds9.dot(pt, x, y, frame=frame, ctype=ctype)
 
+
 def setRaDec(wcs, sourceSet):
     """Set the ra/dec fields in a sourceSet from [XY]Astrom"""
-    
+
     for s in sourceSet:
         s.setRaDec(wcs.pixelToSky(s.getXAstrom(), s.getYAstrom()))
 
+
 def writeSourceSet(sourceSet, outfile="-"):
     if outfile == "-":
         fd = sys.stdout
     else:
         fd = open(outfile, "w")
-    
+
     for s in sourceSet:
-        print >> fd, s.getId(), s.getXAstrom(), s.getYAstrom(), s.getRa(), s.getDec(), s.getPsfFlux(), s.getFlagForDetection()
+        print >> fd, s.getId(), s.getXAstrom(), s.getYAstrom(), s.getRa(
+        ), s.getDec(), s.getPsfFlux(), s.getFlagForDetection()
+
```

Comment:
No. It broke at the wrong place and it probably didn't need to break at all for 110 character limit.